README.md

```diff
@@ -137,6 +137,11 @@ $ kubectl exec -it samplepod -- ip a
 - type (string, required): &quot;multus&quot;
 - kubeconfig (string, optional): kubeconfig file for the out of cluster communication with kube-apiserver. See the example [kubeconfig](https://github.com/intel/multus-cni/blob/master/doc/node-kubeconfig.yaml)
 - delegates (([]map,required): number of delegate details in the Multus
+<<<<<<< Updated upstream
```

Comment:
Is this message a git log? Could you please double check, as well as line 142 and 144.

Reply:
Oups! sorry about that... :)